WordPress/src/test/java/org/wordpress/android/ui/reader/discover/ReaderPostMoreButtonUiStateBuilderTest.kt

```diff
@@ -309,6 +323,34 @@ class ReaderPostMoreButtonUiStateBuilderTest : BaseUnitTest() {
         assertThat(menuItems.find { it.type == ReaderPostCardActionType.SPACER_NO_ACTION }).isNotNull
     }
 
+    @Test
+    fun `does contain block user action for wpcom sites`() = test {
+        // Arrange
+        val post = init()
+        whenever(readerUtilsWrapper.isSelfHosted(post.authorId)).thenReturn(false)
+
+        // Act
+        val menuItems = builder.buildMoreMenuItems(post, dummyOnClick)
+        // Assert
+        assertThat(menuItems.find {
+            it.type == ReaderPostCardActionType.BLOCK_USER
+        }).isNotNull()
+    }
+
+    @Test
+    fun `does not contain block user action for wpcom sites`() = test {
```

Comment:
I guess this test name is meant to be `does not contain block user action for self-hosted sites`?

Reply:
Good catch @mkevins 👍 
Fixed with 26ca9ed9bbf8c9906e3998afd4d884a88eedc43a